kogito-timer-quarkus/README.md

```diff
@@ -169,11 +262,22 @@ You need to download the job service and start it locally
 java -Dquarkus.http.port=8085 -jar target/jobs-service-{version}-runner.jar
 ```
 
+* After Starting Kogito Web Service you should see a similar Log as follows
+
+<p align="center"><img src="docs/images/kogitoWebServiceLog.png"></p>
+
+
 In case you'd like to run the job service with enabled persistence then start 
 Infinispan server before and then run the job service with following command
```

Comment:
How do you start Infinispan server ?
Link ?

Reply:
fixed @radtriste